rules/windows/process_creation/win_apt_bluemashroom.yml

```diff
@@ -15,9 +15,9 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\regsvr32*\AppData\Local\\*'
-            - '*\AppData\Local\\*,DllEntry*'
+        CommandLine|contains:
+            - '\regsvr32*\AppData\Local\\'
+            - '\AppData\Local\\*,DllEntry'
```

Comment:
well, I would better ask the author what he was meant by these `*` in the patterns..

Reply:
fixed